package/skelcd-control-openSUSE.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Thu Jul 30 16:57:34 UTC 2020 - Richard Brown <rbrown@suse.de>
+
+- Remove tmp subvolume for all system roles [boo#1173461][jsc#PM-1898]
```

Comment:
please keep line length limit

Reply:
Why? There's countless lines longer in that changelog already?

Reply:
sysrich has a point.